code/datums/components/plumbing/_plumbing.dm

```diff
@@ -226,8 +217,11 @@
 	active = TRUE
 
 	var/atom/movable/AM = parent
```

Comment:
and a better var name for this too, because you're using it you may as well improve it while you're here